code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm

```diff
@@ -4,7 +4,14 @@
 		if(last_noise <= (world.time) && aggro_noise)
 			playsound(src.loc, list(aggro_noise), 120, 1)
 			last_noise = world.time + 30 SECONDS
-		alpha = 255
+		animate(src, alpha = chameleon_skill, time = 2)
+		sleep(2)
```

Comment:
Its less performance heavy to do spawn(2)
iirc

Reply:
sleep() is used to pause the proc itself. spawn() doesn't pause the proc.